src/main/java/com/thesamet/gerrit/plugins/saml/SamlWebFilter.java

```diff
@@ -235,9 +235,9 @@ private String getUserName(SAML2Profile user) {
         private AuthenticatedUser user;
 
         public AuthenticatedHttpRequest(HttpServletRequest request,
-                                        @Nonnull AuthenticatedUser user) {
+                                        AuthenticatedUser user) {
             super(request);
-            this.user = user;
+            Preconditions.checkNotNull(this.user = user);
```

Comment:
Because I don't know what was the intention of Nonnull in the first place, can't say if this is the right fix or not.